.github/release-drafter.yml

```diff
@@ -0,0 +1,89 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+name-template: "v$RESOLVED_VERSION"
+tag-template: "v$RESOLVED_VERSION"
```

Comment:
Our practice is to used just `$RESOLVED_VERSION` as a tag and release name. However we don't use release-drafter at the moment and it looks like a good idea!

Reply:
fix !